### PR TITLE
Fix go-get bug

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by msrb** [comment]
Tests fail because of new pydocstyle: https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/602

File: runtest.sh [hunk 1]
```diff
@@ -96,6 +96,7 @@ echo "Starting test suite"
 docker run -t \
   -v "${here}:/f8a_worker:ro,Z" \
   --network ${DOCKER_NETWORK} \
+  -u 9007 \
```
Comment (abs51295) [inline runtest.sh]: Any reason why this was added and the chunk from `Dockerfile.tests` was removed?

Comment (msrb) [reply runtest.sh]: The code runs under random non-existent user in OpenShift. We were explicitly creating a user in `Dockerfile.tests`. That's why the tests didn't catch this bug when it was introduced. With this change we are running the tests as a user with ID 9007, but there is no user with such ID in `/etc/passwd`.

Comment (abs51295) [reply runtest.sh]: Makes sense. Thanks for the explanation 👍

**Comment by abs51295** [comment]
[test]

**Pull request merged**